packages/ember-template-compiler/lib/plugins/extract-pragma-tag.js

```diff
@@ -0,0 +1,26 @@
+const PRAGMA_TAG = 'use-component-manager';
```

Comment:
How does this play with https://github.com/ember-cli/ember-cli-htmlbars/pull/114?  It seems like only one or the other should be required, not both.